x/auth/types/stdsignmsg.go

```diff
@@ -3,25 +3,26 @@
 import (
 	"github.com/cosmos/cosmos-sdk/codec/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/legacy/legacytx"
 )
 
 var _ types.UnpackInterfacesMessage = StdSignMsg{}
 
 // StdSignMsg is a convenience structure for passing along a Msg with the other
 // requirements for a StdSignDoc before it is signed. For use in the CLI.
 type StdSignMsg struct {
```

Comment:
This should go in the same legacy package as `StdTx`, etc

Reply:
moved